data/selinux/snappy.te

```diff
@@ -468,6 +469,7 @@ allow snappy_confine_t etc_t:file mounton;
 allow snappy_confine_t home_root_t:dir mounton;
 allow snappy_confine_t ifconfig_var_run_t:dir mounton;
 allow snappy_confine_t modules_object_t:dir mounton;
+allow snappy_confine_t lib_t:dir mounton;
```

Comment:
Is this enough for this functionality to work? It _looks_ okay, but I’m not sure if `mounton` implies that you can read the stuff inside once it is mounted inside. We did this with several other types, so…

Reply:
Thanks! It seems to be enough, our tests are happy but maybe they are not deep enough - I am inclined to use as is and let @stolowski review later and tweak policy and tests if needed.